admin_guide/out_of_resource_handling.adoc

```diff
@@ -184,12 +210,44 @@ eviction signal.
 |`*MemoryPressure*`
 |`*memory.available*`
 |Available memory on the node has satisfied an eviction threshold.
+
+|`*DiskPressure*`
+|`*nodefs.available*`, `*nodefs.inodesFree*`, `*imagefs.available*`, or `*imagefs.inodesFree* 
+|Available disk space and inodes on either the node’s root filesytem or image filesystem has satisfied an eviction threshold.
 |===
 
 When the above is set the node continues to report node status updates at the
 frequency specified by the `node-status-update-frequency` argument, which
 defaults to *10s*.
 
+[[out-of-resource-reclaiming-node-level-resources]]
+=== Reclaiming node level resources
+
+If an eviction threshold has been met and the grace period has passed, the node
+will initiate the process of reclaiming the pressured resource until it has
+observed the signal has gone below its defined threshold.
```

Comment:
might want to add that the node is unschedulable until the pressure is resolved.